flow/project.py

```diff
@@ -3382,20 +3598,20 @@ def completed_operations(self, job):
             if op._complete((job,)):
                 yield name
 
-    def _job_operations(self, job, ignore_conditions=IgnoreConditions.NONE):
-        "Yield instances of _JobOperation constructed for specific jobs."
-        for name in self.operations:
-            group = self._groups[name]
-            yield from group._create_run_job_operations(entrypoint=self._entrypoint, jobs=(job,),
-                                                        default_directives=dict(),
-                                                        ignore_conditions=ignore_conditions,
-                                                        index=0)
+    def _job_operations(self, jobs, group, ignore_conditions=IgnoreConditions.NONE):
```

Comment:
If this is only used in the deprecated function `next_operations` we can remove this and just put the code there.

Reply:
Good idea. Done.